packaging/suse/amazon-ecs-init.spec

```diff
@@ -0,0 +1,93 @@
+#
+# spec file for package amazon-ecs-init
+#
+# Copyright (c) 2015 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+%define short_name amazon-ecs
+Name:           amazon-ecs-init
+Version:        1.4.0
+Release:        0
+Summary:        Amazon EC2 Container Service Initialization
+License:        Apache-2.0
+Group:          System Environment/Base
+Url:            https://github.com/aws/amazon-ecs-init
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{short_name}.service
+Patch1:         cgroup_location.patch
+BuildRequires:  go
+BuildRequires:  systemd
+Requires:       docker >= 1.6.0
+Requires:       systemd
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+ExclusiveArch:  %ix86 x86_64
+
+%description
+The Amazon Container Service initialization will start the ECS agent.
```

Comment:
On 09/27/2015 04:09 PM, Samuel Karp wrote:
>> +Summary:        Amazon EC2 Container Service Initialization
>> +License:        Apache-2.0
>> +Group:          System Environment/Base
>> +Url:            https://github.com/aws/amazon-ecs-init
>> +Source0:        %{name}-%{version}.tar.gz
>> +Source1:        %{short_name}.service
>> +Patch1:         cgroup_location.patch
>> +BuildRequires:  go
>> +BuildRequires:  systemd
>> +Requires:       docker >= 1.6.0
>> +Requires:       systemd
>> +BuildRoot:      %{_tmppath}/%{name}-%{version}-build
>> +ExclusiveArch:  %ix86 x86_64
>> +
>> +%description
>> +The Amazon Container Service initialization will start the ECS agent.
> 
> Missed this before.  Can you change "Amazon Container Service" to "Amazon EC2 Container Service"?
> 

Done


-- 
Robert Schweikert                   MAY THE SOURCE BE WITH YOU
Public Cloud Architect                         LINUX
rjschwei@suse.com
IRC: robjo